### PR TITLE
Added a short note about the HDEE sunpy proposal

### PR DESCRIPTION
**Pull request by wafels**
Let me know what you think about this.

### CODE REVIEW

**Comment by Cadair** [comment]
Your build failiure seems real:

> 8:25:22 PM: Warning, treated as error:
> 8:25:22 PM: /opt/build/repo/posts/2020/2020-06-03_hdee_sunpy.rst:17:Unexpected indentation.

File: posts/2020/2020-06-03_hdee_sunpy.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+SunPy awarded a NASA HDEE Grant
+===============================
+
+.. post:: June 3 2020
+   :author: Jack Ireland
+   :tags: sunpy
+   :category: Release
```
Comment (nabobalis) [inline posts/2020/2020-06-03_hdee_sunpy.rst:7]: I wonder if we have a better cateogry for this.

File: posts/2020/2020-06-03_hdee_sunpy.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+A NASA Heliophysics Data Environment Enhancement (HDEE) grant has been
+awarded to a proposal submitted to support SunPy development.  The
```
Comment (nabobalis) [inline posts/2020/2020-06-03_hdee_sunpy.rst:10]:
```suggestion
awarded to a proposal submitted to support SunPy development. The
```

File: posts/2020/2020-06-03_hdee_sunpy.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+grant was awarded through NASA's 2019 ROSES grant solicitation.
+
+
+The grant supports the following development goals::
```
Comment (nabobalis) [inline posts/2020/2020-06-03_hdee_sunpy.rst:14]:
```suggestion
The grant supports the following development goals:
```

File: posts/2020/2020-06-03_hdee_sunpy.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+* Provide the ability to read spectroscopic data into a spectral data
+  object, thereby enabling its later scientific analysis.
+* Implement a number of heliophysical coordinate systems using the
+  existing SunPy and Astropy-based coordinate system framework.
+* Create example code snippets that use SunPy and packages from the
+  Python in Heliophysics Community; these examples will be shared via
+  the Python in Heliophysics Community.
+
+The PI of the grant is Jack Ireland, and the co-I of the grant is Andy
+Terrel (NumFOCUS).  The grant has a one year duration.  The bulk of
```
Comment (nabobalis) [inline posts/2020/2020-06-03_hdee_sunpy.rst:29]:
```suggestion
Terrel (NumFOCUS). The grant has a one year duration. The bulk of
```

File: posts/2020/2020-06-03_hdee_sunpy.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+the code development will be undertaken by a developer hired through
+NuMFOCUS.  We look forward to supporting the development of SunPy
```
Comment (nabobalis) [inline posts/2020/2020-06-03_hdee_sunpy.rst:31]:
```suggestion
NuMFOCUS. We look forward to supporting the development of SunPy
```

File: posts/2020/2020-06-03_hdee_sunpy.rst (hunk 1 above)
Comment (nabobalis) [inline posts/2020/2020-06-03_hdee_sunpy.rst:7]:
```suggestion
   :category: Update
```

File: posts/2020/2020-06-03_hdee_sunpy.rst [hunk 6]
```diff
@@ -0,0 +1,34 @@
+SunPy awarded a NASA HDEE Grant
+===============================
+
+.. post:: June 3 2020
```
Comment (nabobalis) [inline posts/2020/2020-06-03_hdee_sunpy.rst:4]:
```suggestion
.. post:: June 5 2020
```

**Comment by nabobalis** [comment]
Thanks Jack.

**Pull request merged**